src/Transports/AbstractTransport.php

```
@@ -0,0 +1,369 @@
+        $alternativeParameter = static::alternativeParameter();
+        foreach ($hierarchy as $transport) {
+            $id = static::make($transport)->get($alternativeParameter);
+            if (! is_null($id)) {
+                return $id;
+            }
+        }
+
+        return null;
+    }
+
+    protected static function cleanRequestHierarchy(array $hierarchy): void
```

PHPStan (2.1.31)

Method Ninja\DeviceTracker\Transports\AbstractTransport::cleanRequestHierarchy() has parameter $hierarchy with no value type specified in iterable type array.
See: https://phpstan.org/blog/solving-phpstan-no-value-type-specified-in-iterable-type

(missingType.iterableValue)